services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/domain/Hanke.kt

```diff
@@ -132,7 +132,7 @@ data class Hanke(
         description =
             "Hanke areas data. At least one alue is required for the hanke to be published."
     )
-    override var alueet = mutableListOf<Hankealue>()
+    override var alueet = mutableListOf<SavedHankealue>()
```

Comment:
I'm not sure what would be a better name for this, but this is also used for updating a hanke. If there is a new alue, this would be a bit misleading name? 

How about the same approach as in hanke:
- interface BaseHankeAlue
- HankeAlue for the domain object
- NewHankeAlue for request dto

Reply:
Naming interfaces or abstract classes as BaseXXX is a bad practice. I'm already regretting using it with Hanke.

The SavedHankealue name is misleading for updating a hanke because the class itself is not well suited for that purpose. There is information the user isn't able to update in the class, especially in the Geometriat attached to it.